templates/layout/parts/head.html.twig

```diff
@@ -48,6 +48,14 @@
    {# prevent robots to index GLPI instances #}
    <meta name="robots" content="noindex, nofollow" />
 
+    {# Custom header tags for plugins #}
+    {% for header_tag in header_tags %}
+        {% for property in header_tag.properties %}
+            <{{ header_tag.tag }} {% for key, value in property %}{{ key }}="{{ value }}" {% endfor %}/>
+        {% endfor %}
```

Comment:
I think the correct version is this one:
```suggestion
        <{{ header_tag.tag }} {% for key, value in header_tag.properties %}{{ key }}="{{ value }}" {% endfor %}/>
```

Reply:
Indeed, we were using an array of tags in the hook call.